src/promptflow-devkit/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -1109,18 +1132,98 @@ def _save(
         if LANGUAGE_KEY in kwargs:
             data[LANGUAGE_KEY] = language
 
-        target_flow_file = target_flow_directory / FLOW_DAG_YAML
```

Comment:
After I merge this https://github.com/microsoft/promptflow/pull/2548, 
please modify it to **FLOW_FLEX_YAML**